brunch-config.js

```diff
@@ -20,7 +23,10 @@ exports.config = {
       // }
     },
     stylesheets: {
-      joinTo: 'css/app.css',
+      joinTo: {
+        'css/app.css': /^(web\/static\/css)|(web\/static\/vendor\/css)/,
+        'css/mobileSurvey.css': /^(web\/static\/mobile_survey)/
```

Comment:
Should we change the route to include css in the mobile_survey as well? `/^(web\/static\/mobile_survey\/css)/`